package.json

```diff
@@ -6,6 +6,11 @@
     "@hotwired/turbo-rails": "^7.2.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
```

Comment:
react-dom was added successfully as a dependency 📝 Please be sure to add babel-loader as a dependency rather than a devDependency so that the project functions effectively 🤓

Reply:
@mateo951 Thanks for that kind suggestion. But I would like to leave babel-loader in devDependency for now as is not a stated requirement, and if in the future should I need it in production will add it there. 👍🏽

Reply:
@kinginthenorthcodez Sadly, it is part of the guidelines I follow to review your project 👀 Kindly make the change so that your project meets the requirements 😎